### PR TITLE
New test for "Return Largest Numbers in Arrays" - checking for negative only numbers

### LINKED ISSUE #13456: Return Largest Numbers in Arrays - should check for negative numbers
Challenge [Return Largest Numbers in Arrays](https://www.freecodecamp.com/challenges/return-largest-numbers-in-arrays).

On the automated tests for passing this exercise, you should also have a test with only negative numbers also, like: largestOfFour([[**-1, -3, -6, -25**], [13, 27, 18, 26], [32, 35, 37, 39], [1000, 1001, 857, 1]]);

If you don't do array.sort and try to find the max in each sub-array, if you init a tempMax with 0 for each sub-array, the test will validate. But that would be fake validate, since the max from first sub-array would return 0.

### PR DESCRIPTION
**Pull request by Jamayk**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13456

#### Description

If you don't go for array.sort and try to find one by one the max in each sub-array, if you init maxTemp (temp max for each sub-array) with 0, you can pass the exercise because all the FCC tests are with positive numbers, but if I had an all negative sub-array, my max would had been 0 for that particular sub, and I would still pass the test.

See an example of wrong code that would pass this exercise:

```
function largestOfFour(arr) {
  var maxArr = [];
  for (var i = 0; i < arr.length; i++) {
    var maxTemp = 0;
    for (var j = 0; j < arr[i].length; j++){
      if (arr[i][j] > maxTemp) { 
        maxTemp = arr[i][j];
      }
    }
    maxArr.push(maxTemp);
  }
  return maxArr;
}

```

Ofc, if I change maxTemp to be 1st number from each sub-array, then this would pass the test, including the new test I added with negative only [-72, -3, -17, -10].

### CODE REVIEW

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting.json [hunk 1]
```diff
@@ -324,7 +324,8 @@
...
-        "assert.deepEqual(largestOfFour([[4, 9, 1, 3], [13, 35, 18, 26], [32, 35, 97, 39], [1000000, 1001, 857, 1]]), [9, 35, 97, 1000000], 'message: <code>largestOfFour([[4, 9, 1, 3], [13, 35, 18, 26], [32, 35, 97, 39], [1000000, 1001, 857, 1]])</code> should return <code>[9, 35, 97, 1000000]</code>.');"
+        "assert.deepEqual(largestOfFour([[4, 9, 1, 3], [13, 35, 18, 26], [32, 35, 97, 39], [1000000, 1001, 857, 1]]), [9, 35, 97, 1000000], 'message: <code>largestOfFour([[4, 9, 1, 3], [13, 35, 18, 26], [32, 35, 97, 39], [1000000, 1001, 857, 1]])</code> should return <code>[9, 35, 97, 1000000]</code>.');",
+      	"assert.deepEqual(largestOfFour([[17, 23, 25, 12], [25, 7, 34, 48], [4, -10, 18, 21], [-72, -3, -17, -10]]), [25, 48, 21, -3], 'message: <code>largestOfFour([[17, 23, 25, 12], [25, 7, 34, 48], [4, -10, 18, 21], [-72, -3, -17, -10]])</code> should return <code>[25, 48, 21, -3]</code>.');"
```
Comment (raisedadead) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting.json]: add one more space before this line of code.

**Review by raisedadead — CHANGES_REQUESTED** [review]
Minor change to spacing. When you have done the change please amend your commit and push.

```sh
git checkout fix/new-test-largest-nrs-in-array
Make your changes
git add --all
git commit --amend --no-edit
git push origin fix/new-test-largest-nrs-in-array --force
```

**Review by Greenheart — APPROVED** [review]
@Jamayk Phew, my solution still passes! :wink:

Looking good!
